src/mixins/extensions/PageNavigate.js

```diff
@@ -3,24 +3,27 @@ export default {
     /* istanbul ignore next */
     pageNavigate() {},
```

Comment:
Could you analize if the empty method could be removed

Reply:
thanks the empty method was removed